app/design-system/design-system-hedvig/src/main/kotlin/com/hedvig/android/design/system/hedvig/Tab.kt

```diff
@@ -0,0 +1,396 @@
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.offset
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateMapOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.snapshots.SnapshotStateMap
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.layout.layout
+import androidx.compose.ui.layout.onPlaced
+import androidx.compose.ui.layout.positionInParent
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.Density
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.hedvig.android.design.system.hedvig.TabDefaults.TabSize
+import com.hedvig.android.design.system.hedvig.TabDefaults.TabSize.Mini
+import com.hedvig.android.design.system.hedvig.TabDefaults.TabStyle
+import com.hedvig.android.design.system.hedvig.TabDefaults.TabStyle.Default
+import com.hedvig.android.design.system.hedvig.TabDefaults.maxItemsInTheRow
+import com.hedvig.android.design.system.hedvig.TabDefaults.maxLines
+import com.hedvig.android.design.system.hedvig.tokens.ColorSchemeKeyTokens.ButtonSecondaryAltResting
+import com.hedvig.android.design.system.hedvig.tokens.ColorSchemeKeyTokens.ButtonSecondaryResting
+import com.hedvig.android.design.system.hedvig.tokens.ColorSchemeKeyTokens.SurfacePrimary
+import com.hedvig.android.design.system.hedvig.tokens.ColorSchemeKeyTokens.TextPrimary
+import com.hedvig.android.design.system.hedvig.tokens.LargeTabTokens
+import com.hedvig.android.design.system.hedvig.tokens.MediumTabTokens
+import com.hedvig.android.design.system.hedvig.tokens.MiniTabTokens
+import com.hedvig.android.design.system.hedvig.tokens.SmallTabTokens
+
+@OptIn(ExperimentalLayoutApi::class)
+@Composable
+fun HedvigTabRow(
+  tabTitles: List<String>,
+  selectedTabIndex: Int,
+  onTabChosen: (index: Int) -> Unit,
+  modifier: Modifier = Modifier,
+  tabSize: TabSize = TabDefaults.defaultSize,
+  tabStyle: TabStyle = TabDefaults.defaultStyle,
+) {
+  val density = LocalDensity.current
+  val currentWidthMap = remember { mutableStateMapOf<Int, Int>() }
+  val currentHeightMap = remember { mutableStateMapOf<Int, Int>() }
+  val currentOffsetMap = remember { mutableStateMapOf<Int, Offset>() }
+  val indicatorOffsetX: Dp by animateDpAsState(
+    targetValue = calculateIndicatorOffsetX(currentOffsetMap, selectedTabIndex, density),
+    animationSpec = tween(easing = LinearEasing),
```

Comment:
This may be a bit tricky, but can you try and look at perhaps Sladan's phone where they have some tab indicator like that on some iPhone app and see how the indicator offset moves? I *suspect* that there is room for something which feels a bit smoother than a LinearEasing, but I definitely can't tell just from the top of my head.
You could also try at random some from here https://github.com/HedvigInsurance/android/blob/7e85f0db5781027e32700587c32c517454c95dac/app/core/core-design-system/src/main/kotlin/com/hedvig/android/core/designsystem/material3/motion/MotionTokens.kt#L24-L33 and see if any of them look nicer?

Reply:
I've looked at it on iPhone, and it feels like FastOutSlowInEasing. So I'm trying FastOutSlowInEasing with duration = 600 millis, I think it's close enough